bootloader/assets/grub.go

```diff
@@ -89,7 +96,7 @@ if [ -e /EFI/ubuntu/grubenv ]; then
 fi
 
 # standard cmdline params
-set cmdline="console=ttyS0 console=tty1 panic=-1"
+set snapd_static_cmdline_args='####STATIC_CMDLINE####'
```

Comment:
this shouldn't be templated, on the contrary we should keep a mapping on the side of edition to static command line, we should have a test that checks that the mapping is correct for the current asset edition

Reply:
but we do keep track of the edition to static command line though with the assets functionality? The way I see it, this templating is just done to simplifiy the implementation of the mapping of edition to static commandline, see this (from this PR): 

https://github.com/snapcore/snapd/blob/ef1e8ab3a81951b942d4a3c0fddbfa1bc47d78e6/bootloader/assets/grub.go#L155-L164